kit/KitQueue.hpp

```diff
@@ -26,7 +26,18 @@ class TilePrioritizer
 {
 public:
     virtual ~TilePrioritizer() {}
-    virtual float getTilePriority(const TileDesc &) const { return 0.0; }
+
+    enum class Priority {
+        NONE = -1,  // an error
```

Comment:
Good this is the right way around - amusingly the core scheduler's priority enum is inverted - so small values are high priority for no really obvious reason ;-)